patches/php-css-parser-commit-10a2501.patch

```diff
@@ -0,0 +1,143 @@
+From fa139f65c5b098ae652c970b25e6eb03fc495eb4 Mon Sep 17 00:00:00 2001
+From: Weston Ruter <weston@xwp.co>
+Date: Wed, 25 Jul 2018 10:38:59 -0700
+Subject: [PATCH 1/2] Fix parsing CSS selectors which contain commas
+
+---
+ .../CSS/RuleSet/DeclarationBlock.php          | 60 ++++++++++++++++++-
+ tests/Sabberworm/CSS/ParserTest.php           |  6 ++
+ tests/files/specificity.css                   |  4 +-
+ 3 files changed, 68 insertions(+), 2 deletions(-)
+
+diff --git a/lib/Sabberworm/CSS/RuleSet/DeclarationBlock.php b/lib/Sabberworm/CSS/RuleSet/DeclarationBlock.php
+index e18f5d8..26c2e12 100644
+--- a/lib/Sabberworm/CSS/RuleSet/DeclarationBlock.php
++++ b/lib/Sabberworm/CSS/RuleSet/DeclarationBlock.php
```

Comment:
Note that SabberWorm PHP CSS Parser seems to use PSR-2/PSR-12, not WPCS, so code style needs to be adapted if this is to be merged into the upstream package.